mlflow/tracking/utils.py

```diff
@@ -90,13 +90,7 @@ def _is_databricks_uri(uri):
 
 
 def _get_file_store(store_uri, **_):
-    return FileStore(file_utils.local_file_uri_to_path(store_uri), store_uri)
-
-
-def _is_database_uri(uri):
-    if urllib.parse.urlparse(uri).scheme not in DATABASE_ENGINES:
-        return False
-    return True
+    return FileStore(store_uri, store_uri)
```

Comment:
`FileStore` itself does the conversion so we don't need `local_file_uri_to_path` here.

Reply:
This same function is repeated multiple places. Can we unify?

Reply:
Unfortunately the MLflowClient one doesn't use a unique artifact repo (reuses `store_uri`) and the server one accepts a different one. Not sure if we want to change that behavior...

Reply:
indeed. but the api could have 2 arguments `store_uri` and `artifact_uri` and the caller can send in the same one. No?